src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -308,7 +309,16 @@ static CliRootCommand CreateCommandLineParser()
 
         return RunAsync(serverConfiguration, cancellationToken);
     });
-    return rootCommand;
+
+    var config = new CliConfiguration(rootCommand)
+    {
+        // By default, SCI will catch all exceptions, log them to the console, and return a non-zero exit code.
```

Comment:
```suggestion
        // By default, System.CommandLine will catch all exceptions, log them to the console, and return a non-zero exit code.
```

As otherwise that's our usual abbreviation for System.Collections.Immutable...